packages/plugin-align/src/Align.ts

```diff
@@ -103,11 +103,11 @@ export class Align<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T
      */
     align(params: AlignParams): void {
         const nodes = params.context.range.targetedNodes().flatMap(node => {
-            if (node.editable) {
+            if (this.editor.isEditable(node)) {
```

Comment:
Another example of why it would be beneficial to think in terms of mode, rather than editor that check if something is editable:
We could introduce `params.ctx.mode.isEditable(node)` even though I can't think of any usefull case right now.

Reply:
lol

Reply:
He's right, actually. Testing whether a node is editable or not through the editor itself (or a plugin, really) considers that only one mode can be active at a time, which might be wrong. Different users might have different modes. A subcommand of a complex command might want to modify the structure in the context of a particular mode.

Reply:
We now have `params.context.range.mode.isEditable(node)` which is roughly equivalent as you can pass a mode to `withRange`.